pkg/collect/host_sysctl.go

```diff
@@ -0,0 +1,52 @@
+package collect
+
+import (
+	"bytes"
+	"encoding/json"
+
+	"github.com/lorenzosaino/go-sysctl"
+	"github.com/pkg/errors"
+	troubleshootv1beta2 "github.com/replicatedhq/troubleshoot/pkg/apis/troubleshoot/v1beta2"
+)
+
+// Ensure `CollectHostSysctl` implements `HostCollector` interface at compile time.
+var _ HostCollector = (*CollectHostSysctl)(nil)
+
+// Path to the kernel virtual files, defaults to /proc/sys
+var sysctlVirtualFiles = sysctl.DefaultPath
+
+const HostSysctlPath = `host-collectors/system/sysctl.json`
+
+type CollectHostSysctl struct {
+	hostCollector *troubleshootv1beta2.HostSysctl
+	BundlePath    string
+}
+
+func (c *CollectHostSysctl) Title() string {
+	return hostCollectorTitleOrDefault(c.hostCollector.HostCollectorMeta, "Sysctl")
+}
+
+func (c *CollectHostSysctl) IsExcluded() (bool, error) {
+	return isExcluded(c.hostCollector.Exclude)
+}
+
+func (c *CollectHostSysctl) Collect(progressChan chan<- interface{}) (map[string][]byte, error) {
+	client, err := sysctl.NewClient(sysctlVirtualFiles)
+	if err != nil {
```

Comment:
There is an error here

```
E1107 12:10:03.235230   56942 host_sysctl.go:39] Failed to initialize sysctl client: could not create client: directory /proc/sys/ does not exist
```

I think we should revert back to calling `sysctl` instead of relying on a library, otherwise we need to handle those rare environments, e.g darwin that don't have `/proc/sys`.

Reply:
Hmm yeah true, darwin won't have `procfs`, however the props will be completely different too and even the output will differ between `sysctl -a` in darwin vs linux. But agree it's probably the best approach out of both options. Gonna go back to that and have a common output from the collector

Reply:
Done, let me know what you think @banjoh